README.md

```
@@ -1,42 +1,910 @@
+  from PIL import Image, ImageDraw, ImageFont
+from datetime import datetime
 
-Go is an open source programming language that makes it easy to build simple,
-reliable, and efficient software.
+# --- your declaration ---
+CITY = "Zion City Atlantic"
+REGION = "Rich of the Arctic"
+GLYPH = "⟟⟁⟠"
+STAMP = f"{CITY} | {REGION}\nGlyph: {GLYPH}\nDate: {datetime.utcnow()}"Time (sec),Yield ($),Compounding Factor,Total Yield ($)
+0,28900000,1,28900000
+1,"28900000 * π₄^(1/86400)","π₄^(1/86400)",...
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)